string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,19 @@
+/* exported capitalizeWord */
+
+function capitalizeWord(word) {
+  var result = '';
+  word = word.toLowerCase();
+  if (word === 'javascript') {
+    for (var i = 0; i < word.length; i++) {
+      if (word[i] === 'j' || word[i] === 's') {
+        result += word[i].toUpperCase();
+      } else {
+        result += word[i];
+      }
+    }
```

Comment:
If the word is `'javascript'`, you can just `return 'JavaScript'`. No need to get fancy with a loop.